cmd/generate/config/rules/onfido.go

```diff
@@ -0,0 +1,26 @@
+package rules
+
+import (
+    "github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+    "github.com/zricethezav/gitleaks/v8/config"
+)
+
+func OnfidoAPIToken() *config.Rule {
+    // define rule
+    r := config.Rule{
+        RuleID:      "onfido-live-api-token",
+        Description: "Found an Onfido live API Token posing a risk of unauthorized access to identity verification services data, potentially leading to misuse of PII.",
+        Regex:       utils.GenerateUniqueTokenRegex(`(?:api_live(?:_[a-zA-Z]{2})?\.[a-zA-Z0-9-_]{11}\.[-_a-zA-Z0-9]{32})`, true),
```

Comment:
A few thoughts on this:

1. The wrapping `(?: ... )` aren't necessary 
2. This could be simplified using `\w`. 
3. I believe you'd also want to set [`IsCaseInsensitive`](https://github.com/gitleaks/gitleaks/blob/4e43d1109303568509596ef5ef576fbdc0509891/cmd/generate/config/utils/generate.go#L58) to false so it doesn't match `API_LIVE`.
```suggestion
        Regex:       utils.GenerateUniqueTokenRegex(`api_live(?:_[a-zA-Z]{2})?\.[\w-]{11}\.[\w-]{32}`, false),
```

Reply:
Hello Richard, thank you for the review!
I wasn't sure if i needed a capture group or not so i opted for the latter; i agree for all your suggestions